Dockerfile

```diff
@@ -17,10 +17,13 @@ RUN mkdir -p /go/src/sig.k8s.io/gateway-api
 WORKDIR /go/src/sig.k8s.io/gateway-api
 COPY  . .
 RUN useradd -u 10001 webhook
-RUN cd cmd/admission/ && CGO_ENABLED=0 GOOS=linux go build -a -o gateway-api-webhook && chmod +x gateway-api-webhook
+ARG TAG
+ARG COMMIT
+RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -a -o gateway-api-webhook \
+      -ldflags "-s -w -X main.VERSION=$TAG -X main.COMMIT=$COMMIT" ./cmd/admission
 
 FROM scratch
-COPY --from=build-env /go/src/sig.k8s.io/gateway-api/cmd/admission/gateway-api-webhook .
+COPY --from=build-env /go/src/sig.k8s.io/gateway-api/gateway-api-webhook .
 COPY --from=build-env /etc/passwd /etc/passwd
```

Comment:
Do you know if/why we need this?

Reply:
I think this is for the user `webhook` that is created in the build stage. This ensures that that the webhook user is present in the final image as well (to run the image in as non-root).

Reply:
This has now been removed.